hugo/content/community.md

```diff
@@ -4,5 +4,6 @@ section: community
 url: community
 title: Community
 socialDescription: Join our Community and contribute and collaborate with other developers
-socialImage: img/logos/logo-bg-white.png
+socialImage: img/logo-bg-blue.png
```

Comment:
in order to use our default image, we can just delete this line asking for a different one :dagger: 
```suggestion
```